### ISSUE TITLE
Non UWP build

### ISSUE THREAD

**Issue by Terricide** [ortclib/ortclib-sdk#2]
Any chance there are plans to be able to use this library in a regular .net application?

**Comment by robin-raymond** [comment]
I'm not sure on the technical details of how to do it. If I could see some examples of how to wrap C++ into a NET application it might be possible. In the next version to come we will be able to generate the C# UWP wrappers but I don't know if we can do the same for .NET.

**Comment by robin-raymond** [comment]
@Terricide yes; it's now under consideration; I've done some testing and it appears to be doable given the progress to support Win32 to wrap it for .NET too;

**Comment by Terricide** [comment]
That is awesome!!! I've been using https://github.com/radioman/WebRtc.NET but I can't get it to work on Windows 7 and I want to support the Edge browser :)

**Comment by Terricide** [comment]
Hows it going were you ever able to get it to work?

**Comment by jamescadd** [comment]
I can report back that others have managed to use the UWP bits via the Desktop Bridge.  We would like to provide a sample/walkthrough of this process.  At the moment a native .NET flavor of WebRTC is not on the roadmap.
